velox/exec/Operator.cpp

```diff
@@ -368,6 +368,30 @@ void Operator::recordSpillStats() {
     common::updateGlobalMaxSpillLevelExceededCount(
         lockedSpillStats->spillMaxLevelExceededCount);
   }
+
+  if (lockedSpillStats->spillReadBytes != 0) {
+    lockedStats->addRuntimeStat(
+        "spillReadBytes",
+        RuntimeCounter{static_cast<int64_t>(lockedSpillStats->spillReadBytes)});
+    common::updateGlobalSpillReadBytes(lockedSpillStats->spillReadBytes);
+  }
+
+  if (lockedSpillStats->spillReadTimeUs != 0) {
+    lockedStats->addRuntimeStat(
+        "spillReadTimeUs",
+        RuntimeCounter{
+            static_cast<int64_t>(lockedSpillStats->spillReadTimeUs)});
+    common::updateGlobalSpillReadTime(lockedSpillStats->spillReadTimeUs);
+  }
+
+  if (lockedSpillStats->spillReadDeserializationTimeUs != 0) {
+    lockedStats->addRuntimeStat(
+        "spillReadDeserializationTimeUs",
+        RuntimeCounter{static_cast<int64_t>(
+            lockedSpillStats->spillReadDeserializationTimeUs)});
```

Comment:
```
s/spillReadDeserializationTimeUs/spillDeserializationTimeUs/
```
If we want to have read prefix, then let's add write prefix for all the write path stats. But please don't change the stats structure definition which might break Prestissimo.
```
spillWriteXXX
spillReadXXX
```

spillReadDesTimeUs
spillWriteSerTimeUs